### PR TITLE
Sign up dims for additional review roles

### PR DESCRIPTION
**Pull request by dims**
sign up for moar reviews in 1.20

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm
/retest

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -222,6 +222,7 @@ aliases:
     - krzyzacy
```
Comment (BenTheElder) [inline OWNERS_ALIASES:222]: we should probably prune this list, in particular @krzyzacy @ixdy

Comment (krzyzacy) [reply OWNERS_ALIASES:222]: feel free to remove me :-)

Comment (dims) [reply OWNERS_ALIASES:222]: will do in another PR @krzyzacy

**Comment by wojtek-t** [comment]
LGTM

File: OWNERS_ALIASES (hunk 1 above)
Comment (BenTheElder) [reply OWNERS_ALIASES:222]: I sent an inactive-googler prune to test-infra, will prune k/k next

Comment (BenTheElder) [reply OWNERS_ALIASES:222]: https://github.com/kubernetes/kubernetes/pull/93980 will probably wind up rebasing ontop of this one later

**Pull request merged**